### DISCUSSION TITLE
Problems finding

### DISCUSSION THREAD

**Discussion by mulano1977** [terminatorul/NvStrapsReBar#105] [Q&A]
Hello, I have a problem adding the driver. (As per instructions from https://github.com/xCuri0/ReBarUEFI/wiki/Adding-FFS-module).
The search in the UEFITool doesn't spit out anything and I can't find an entry for DXEdriver either -.- Has anyone managed to do it successfully on a Gigabyte Z390 Gaming X v1.0? If so, I would be grateful for step-by-step instructions.

My system:
MB: Gigabyte Z390 Gaming X
GC: MSI RTX 2080 Gaming X Trio (MS-V372)

![UEFITool 0 28 0 - Z390GAMINGX F11 25 08 2024 13_02_19](https://github.com/user-attachments/assets/ff8f46a5-dee6-4113-b3dd-e6e0c476f354)

**Okay, update:**

I found it and added it successfully.

![UEFITool 0 28 0 - Z390GAMINGX F11a rom 25 08 2024 14_14_08](https://github.com/user-attachments/assets/60a6aaa1-12f7-4f11-8938-b72cccf2bdd5)

Now I wanted to flash the new BIOS in the BIOS using Qflash.
Unfortunately, it says "invalid BIOS". Now my question is:
I have now created a bootable stick with the modified Efiflash (according to the instructions for Gigabyte https://github.com/xCuri0/ReBarUEFI/wiki/Flashing-modified-UEFI). Can I flash without any problems? I have a dual BIOS and hope that the BIOS backup will take effect if it fails.

**### update2:**

Unfortunately, flashing with the modified Efiflash didn't work.

I used [FPT-Flash](https://winraid.level1techs.com/t/flashing-gigabyte-while-avoiding-invalid-bios-image/31185/305) for this in case anyone else has the problem.

**It is important to mention that FPT-Flash overwrites the BIOS without regard to losses, even if it is a BIOS from a different motherboard.**

Conclusion: with FPT-Flash (Win64 version v12) it worked without any problems. Perhaps it should be added to the list of problems.
![yeah](https://github.com/user-attachments/assets/29f069f9-f357-48b1-b638-5ce9608601d3)

**Comment by mirh** [comment]
Bios [F10i](https://github.com/user-attachments/files/19606161/mb_bios_z390_gaming_x_f10i.zip) is probably the one you want, from right before they introduced the capsule annoyance.
It still needs to be installed with [modded efiflash](https://winraid.level1techs.com/t/flashing-gigabyte-while-avoiding-invalid-bios-image/31185/248) though (which for some reason took me 1-2 hours to complete the write process, but still).
Thanks to inspire me not to care about the uefitool warnings!
